libs/application/templates/new-primary-school/src/fields/CurrentSchool/CurrentSchool.tsx

```diff
@@ -0,0 +1,39 @@
+import { FieldBaseProps } from '@island.is/application/types'
+import { FC } from 'react'
+import { CategoryCard, Link, Icon, Text, Box } from '@island.is/island-ui/core'
+import * as styles from './CurrentSchool.css'
+import { useLocale } from '@island.is/localization'
+import { newPrimarySchoolMessages } from '../../lib/messages'
+
+export const CurrentSchool: FC<FieldBaseProps> = ({ application }) => {
+  const { formatMessage } = useLocale()
+  // TODO: Use data from JÚNÍ
```

Comment:
Todo statement?

Reply:
Removed component. We don't have all the information that we need for the CurrentSchool component